### PR TITLE
Performance: Freeze string literal

### PR DESCRIPTION
**Pull request by johnnyshields**
Add adds Ruby 2.4 to Travis

### CODE REVIEW

**Comment by johnnyshields** [comment]
@CloCkWeRX this is ready to merge

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,16 @@
...
   global:
     - BROWSER=chrome
   matrix:
-    - DB=mysql
     - DB=postgres
-    - DB=sqlite
-
-matrix:
-  exclude:
-    - rvm: 2.3.3
```
Comment (CloCkWeRX) [inline .travis.yml]: Probably better to just update this exclude to 2.3.6, ditto below for the unlikely chance we bring in mysql/sqlite regression

**Comment by johnnyshields** [comment]
@CloCkWeRX that's fine, let's go ahead and release 0.16.0 then as I have a lot more improvements in the pipeline.

**Comment by johnnyshields** [comment]
FYI I'm using the current master in production now.

**Comment by CloCkWeRX** [comment]
Weird. Travis builds increased by an hour after this:
https://travis-ci.org/fatfreecrm/fat_free_crm/builds

Hopefully, its just something on the underlying travis infrastructure

**Comment by CloCkWeRX** [comment]
Ah, we swapped from exclude to allow failures, meaning run time went way up.

**Pull request merged**